content/applications/services/helpdesk.rst

```diff
@@ -5,20 +5,19 @@
 Helpdesk
 ========
 
-Odoo *Helpdesk* is a ticketing-based customer support application. Multiple teams can be configured
-and managed in one dashboard, each with their own pipeline for tickets submitted by customers.
-Pipelines are organized in customizable stages that enable teams to track, prioritize, and solve
-customer issues quickly and efficiently.
+Odoo **Helpdesk** is a ticketing-based customer support application. Multiple teams can be
+configured and managed in one dashboard, each with their own pipeline for tickets submitted by
+customers. Pipelines are organized in customizable stages that enable teams to track, prioritize,
+and solve customer issues quickly and efficiently.
 
 Create a Helpdesk team
 ======================
 
-To view or modify *Helpdesk* teams, go to :menuselection:`Helpdesk app --> Configuration -->
+To view or modify **Helpdesk** teams, go to :menuselection:`Helpdesk app --> Configuration -->
 Helpdesk Teams`. To create a new team, click the :guilabel:`New` button in the top-left of the
 dashboard.
 
 .. image:: helpdesk/helpdesk-teams-list.png
-   :align: center
    :alt: View of the Helpdesk teams page in Odoo Helpdesk.
 
 On the blank Helpdesk team form, enter a :guilabel:`Name` for the new team. Then, enter a
```

Comment:
It's not letting me suggest directly, but in line 24, it says, "...in the field below the team name," - does this field have a name that can be used with :guilabel:?

Also, in the 'important' admonition block, I would probably revise lines 28-29 for conciseness:

The team description is published on the public-facing :doc:`ticket submission form
   <helpdesk/overview/receiving_tickets>`.

Reply:
No, the field does not have a guilabel